src/frontends/pytorch/src/op/matmul.cpp

```diff
@@ -0,0 +1,21 @@
+ #include "openvino/frontend/pytorch/node_context.hpp"
+#include "pt_framework_node.hpp"
+#include "utils.hpp"
+
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+using namespace ov::op;
+OutputVector translate_mv(const NodeContext& context) {
+    num_inputs_check(context, 3, 7);  
+    auto matrix = context.mark_node(std::make_shared<v0::Convert>(context.get_input(0), element::f32));
+    auto vector = context.mark_node(std::make_shared<v0::Convert>(context.get_input(1), element::f32));
+    auto result = context.mark_node(std::make_shared<v1::Multiply>(matrix, vector));  // Custom operation: element-wise multiplication
```

Comment:
according to openvino [docs](https://docs.openvino.ai/2023.3/openvino_docs_ops_arithmetic_Multiply_1.html), Multiply operation performs element-wise multiplication of 2 tensors using basic arithmetic rules and broadcasting their shapes, but [torch.mv](https://pytorch.org/docs/stable/generated/torch.mv.html#torch-mv) is dot product of matrix and vector, it means that it is following to matrix multiplication rules. The result thesef operations will be different.